### PR TITLE
Add a few border-related CSS properties

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds:

* `border-top-width`
* `border-top-style`
* `border-top-right-radius`

The last one is my first encounter with subfeatures, which raised a couple questions that I couldn't find answers for in the docs.

1. On subfeatures, should notes which apply only to a prefix also include the prefix and version numbers in the subfeature? The source table on MDN doesn't do that (so I didn't do it here), but I'm not sure how much specificity is desirable in this repo.
2. Should notes be edited to remove mention of software which is out-of-scope for this repo? For example, this PR mentions Gecko, Thunderbird, and SeaMonkey. Should it be edited down to refer to Firefox only? Or is retaining such information from the MDN wiki desirable?

### CODE REVIEW

File: css/properties/border-top-width.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "css": {
+    "properties": {
+      "border-top-width": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-top-width",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
```
Comment (Elchi3) [inline css/properties/border-top-width.json]: This should be "android_webview". I need to update the versions we've collected to talk about the weird android versions before android started to use chromium versions.

File: css/properties/border-top-right-radius.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+              },
+              {
+                "alternative_name": "-moz-border-radius-topright",
+                "version_added": "1",
+                "version_removed": "12"
+              }
+            ],
+            "firefox_android": {
+              "version_added": true,
+              "notes": "Prior to Gecko 50.0 (Firefox 50.0 / Thunderbird 50.0 / SeaMonkey 2.47), border styles of rounded corners were always rendered as if <code>border-style</code> was solid. This has been fixed in Gecko 50.0."
```
Comment (Elchi3) [inline css/properties/border-top-right-radius.json]: Leave out Gecko, Thunderbird and SeaMonkey. Just `"Prior to Firefox 50,"`

File: css/properties/border-top-right-radius.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "4",
+                "notes": "Prior to Gecko 50.0 (Firefox 50.0 / Thunderbird 50.0 / SeaMonkey 2.47), border styles of rounded corners were always rendered as if <code>border-style</code> was solid. This has been fixed in Gecko 50.0."
```
Comment (Elchi3) [inline css/properties/border-top-right-radius.json]: Leave out Gecko, Thunderbird and SeaMonkey. Just `"Prior to Firefox 50,"`

**Comment by Elchi3** [comment]
>  On subfeatures, should notes which apply only to a prefix also include the prefix and version numbers in the subfeature? The source table on MDN doesn't do that (so I didn't do it here), but I'm not sure how much specificity is desirable in this repo.

I think the way you've done this here is great. Imo, there is not really a need to put the prefix again for the subfeature.

>  Should notes be edited to remove mention of software which is out-of-scope for this repo? For example, this PR mentions Gecko, Thunderbird, and SeaMonkey. Should it be edited down to refer to Firefox only? Or is retaining such information from the MDN wiki desirable?

Yes, MDN shifted from documenting Gecko products to documenting Open Web technologies, but there is (legacy) Gecko product family information all around still. For the Web, only Firefox matters. This project will clean up a lot of that.

**Comment by ddbeck** [comment]
OK, I've fixed the webview problem (and went back and fixed it on an property I messed up on my last PR) and ditched the non-web version numbers. Let me know if you want to see any other changes. Thanks!

**Pull request merged**